### ISSUE TITLE
Data loss: not an sstable (bad magic number):

### ISSUE THREAD

**Issue by jeremy-rutman** [balancap/SSD-Tensorflow#241]
on running `python3 eval_ssd_network.py     --eval_dir=${EVAL_DIR}     --dataset_dir=${DATASET_DIR}     --dataset_name=pascalvoc_2007     --dataset_split_name=test     --model_name=ssd_300_vgg     --checkpoint_path=${CHECKPOINT_PATH}     --batch_size=1`

I hit 

> `Could not open ./checkpoints/VGG_VOC0712_SSD_300x300_ft_iter_120000.ckpt.data-00000-of-00001: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
`

anyone have a clue?

**Comment by WangJerry95** [comment]
The ${CHECKPOINT_DIR} should be "./checkpoints/VGG_VOC0712_SSD_300x300_ft_iter_120000.ckpt" in your case.

**Comment by jeremy-rutman** [comment]
indeed, it is.  The problem appears to be some file format difference.
